dotnet/src/SemanticKernel/CoreSkills/HttpSkill.cs

```diff
@@ -67,13 +61,8 @@ public async Task<string> GetAsync(string uri)
     /// <returns>The response body as a string.</returns>
     [SKFunction("Makes a POST request to a uri")]
     [SKFunctionContextParameter(Name = "body", Description = "The body of the request")]
-    public async Task<string> PostAsync(string uri, SKContext context)
-    {
-        using var httpContent = new StringContent(context["body"]);
-        var response = await this._client.PostAsync(uri, httpContent);
-        var content = response.Content;
-        return await content.ReadAsStringAsync();
-    }
+    public Task<string> PostAsync(string uri, SKContext context) =>
```

Comment:
While we're updating, could you please plumb through the CancellationToken (accessible from SKContext.CancellationToken) to the downstream async calls? Same for Get, Put, Delete.

Reply:
Happy to.  I actually had it on my list of questions / feedback how cancellation was intended to be handled... I hadn't seen yet it was part of the context.